build.py

```diff
@@ -92,44 +91,10 @@ def overlapFlag(varFont):
         # Set OVERLAP_SIMPLE bit for simple glyphs
         elif glyph.numberOfContours > 0:
             glyph.flags[0] |= 0x40
-    return varFont
-
-def build_font_instance(instance, *steps):
-    if os.path.exists(OUTPUT_DIR / "static") == False:
-        os.mkdir(OUTPUT_DIR / "static")
-
```

Comment:
Build fails w/o this -- let's move it up to `__main__`!

Reply:
Oh whoopsy. Yeah, that should be kept.

Reply:
I added the check back into the build script now where the static instances are built.